src/RLogin.tsx

```diff
@@ -121,6 +121,10 @@ export class RLogin {
     this.updateState({ show: true })
   }
 
+  public showModalInfo = () => {
+    this.updateState({ show: true, currentStep: 'ConfirmInformation', displayMode: true })
```

Comment:
Maybe we can add a `currentStep = 'ShowWalletInfo'` that handles this inside the core. This is like touching the internal state.

Reply:
nice!, I made that changes in the PR #177